### PR TITLE
Salt Day 13

### PR DESCRIPTION
**Pull request by Parzivus**
Not too many changes this time either. We're almost there!

### CODE REVIEW

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 1]
```diff
@@ -665,7 +665,7 @@ SHIKI was being kept locked up somewhere by Tohno Makihisa,
 [sha:dde1db232f9e8078f37f2e1f82cb82878dcee161]{
 -- Page 129, Offset 32028.
 -- 「……シキにとって、俺は自分の名前を使って遠野シキになりすましているニセモノなわけか」
-"...So to SHIKI, I'm just an impostor who's using his name to impersonate him."
+"...So, to SHIKI, I'm just an impostor who's using his name to impersonate him."
```
Comment (yk1512) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:668]: I wouldn't add a comma here

File: script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt [hunk 2]
```diff
@@ -38,7 +38,7 @@ She propels herself along like a creeping shadow.
 [sha:f18bf0a12cfd992d33b84f09084f6350d03e2fd8]{
 -- Page 260, Offset 32383.
 -- 物音ひとつ立てないのは当然として、自らの気配……匂い、体温、鼓動といった生体反応すら外界に漏らしていない。
-Her advance is silent of course but that's not all―――her scent, her body heat, her heartbeat...
+Her advance is silent, of course, but that's not all―――her scent, her body heat, her heartbeat...
```
Comment (yk1512) [inline script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt:41]: Remove first comma.

> Her advance is silent of course, but that's not all

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 13/13_00_ARC13_4.txt [hunk 3]
```diff
@@ -529,8 +529,9 @@ And yet,
 [sha:d79014ed0e0f96e98244f878af98a0d34ed2effc]{
 -- Page 337, Offset 32596.
 -- 「おまえたちは死を拒む事でその強大な力を得るが、同時に弱さをも作り上げた。
-"By rejecting death you people have obtained incredible power,
- but at the same time you have hamstrung yourselves.
+// Too much?
+"By rejecting death, you people have obtained incredible power,
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_4.txt:533]: I'd probably drop the one after `death,` the one after `time` can probably stay

File: script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt [hunk 4]
```diff
@@ -319,7 +321,7 @@ The school building shudders and shakes.
 [sha:227d2577f98a5d10e3997bdb2caa8bc094bfa9f5]{
 -- Page 278, Offset 32437.
 -- 大気に充満した魔力が連鎖起爆し、校舎の背骨にあたる鉄骨をきしませる。
-Its steel frame groans under the stress caused by the magical energy that had filled the air detonating in a chain reaction.
+Its steel frame groans under the stress caused by the magical energy that had filled the air, detonating in a chain reaction.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt:324]: I think this comma incorrectly partitions the sentence. The stress on the building is from the magical energy detonating, but with this comma it kinda reads like the steel frame has detonated in a chain reaction

File: script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt [hunk 5]
```diff
@@ -523,6 +525,7 @@ The enormous left arm goes against the entire concept of that Ghoul.
 [sha:a1dd2b68a79c670586757e846b4657caefe38d53]{
 -- Page 292, Offset 32476.
 -- 全体のバランスを崩し、移動にも悪影響を与えている。
+// Can we check whether off-balance is more common with or without a hyphen?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt:528]: No hyphen
2022-06-03-094524_1306x509_scrot

**Review by rschlaikjer — APPROVED** [review]

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 6]
```diff
@@ -785,7 +785,7 @@ But even putting all that aside, I still can't leave Roa to his own devices.
 [sha:d34ca1cd632eeb27cdbdb8364945ad3806759cb0]{
 -- Page 137, Offset 32051.
 -- ……うん。だから俺も急がないと。最後まで手伝うって、あいつと約束したんだから」
-...Yeah. That's why I need to get going. After all, I promised that I would help her to the very end."
+...Yeah. That's why I need to get going. After all, I promised that I would help her until the very end."
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:788]: not a fan of this change. "to the very end" is a phrase. Much more kakkoi

File: script/Arcueid/Day 13/13_00_ARC13_4.txt [hunk 7]
```diff
@@ -88,7 +88,7 @@ I've almost caught up.
 If I make it in time, we still have a chance.
  Arcueid knows about my eyes, and she's a calm and rational thinker on the battlefield.
  Once she knows I'm here, she'll let me make full use of my abilities.
- She'll defend and I'll attack, same way we played it against Vlov.
+ She'll defend and I'll attack, the same way we played it against Vlov.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_4.txt:91]: I'll attack: same way

File: script/Arcueid/Day 13/13_00_ARC13_4.txt (hunk 3 above)
Comment (Avuxo) [reply script/Arcueid/Day 13/13_00_ARC13_4.txt:533]: I disagree, I like both commas personally.

File: script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt [hunk 8]
```diff
@@ -92,7 +92,7 @@ Ciel mechanically advances on the blind spot that will spell Roa's inevitable de
 [sha:5630be87ba37db84f2a112f43efcdfb3c0f80edc]{
 -- Page 264, Offset 32393.
 -- 周囲の情報は感知魔術によって把握していた。
-Through Perception Magecraft she can sense what's going on around her.
+Through Perception Magecraft, she can sense what's going on around her.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt:95]: revert

**Review by Avuxo — APPROVED** [review]
lgtm pending changes

**Pull request merged**